usersync/cookie.go

```diff
@@ -20,70 +19,140 @@ const uidTTL = 14 * 24 * time.Hour
 
 // Cookie is the cookie used in Prebid Server.
 //
-// To get an instance of this from a request, use ParseCookieFromRequest.
-// To write an instance onto a response, use SetCookieOnResponse.
+// To get an instance of this from a request, use ReadCookie.
+// To write an instance onto a response, use WriteCookie.
 type Cookie struct {
-	uids   map[string]uidWithExpiry
+	uids   map[string]UIDEntry
 	optOut bool
 }
 
-// uidWithExpiry bundles the UID with an Expiration date.
-// After the expiration, the UID is no longer valid.
-type uidWithExpiry struct {
+// UIDEntry bundles the UID with an Expiration date.
+type UIDEntry struct {
```

Comment:
Why was `uidWithExpiry` renamed to `UIDEntry`? Isn't the latter more descriptive? If we decide to leave `UIDEntry` shouldn't we drop the `Entry` suffix? I'm not great at naming.

Reply:
Good question, I'm split between naming it `UID` vs. `UIDEntry`, let's bring it up at next meeting